jira/client.py

```diff
@@ -325,7 +327,7 @@ class JIRA(object):
     AGILE_BASE_URL = GreenHopperResource.AGILE_BASE_URL
 
     def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=None, kerberos=False, kerberos_options=None,
-                 validate=False, get_server_info=True, async_=False, logging=True, max_retries=3, proxies=None,
+                 validate=False, get_server_info=True, async_=False, async_workers=2, logging=True, max_retries=3, proxies=None,
```

Comment:
Why default to `2`?

Reply:
@dbaxa because the current library *already* defaults to 2, I'm just being conservative about changing a library's expected behaviour. I always set this to 5 and I think it's a reasonable setting, but I'm erring on preserving the current behaviour and allowing users to override it.

Reply:
@dchevell sure but just because the library default to 2 isn't a good reason to keep it at 2. Changing the worker count to 5 is probably going to be more useful / closer to what people who use the async code want.